packages/flutter_tools/lib/src/cache.dart

```diff
@@ -1102,18 +1102,42 @@ class IosUsbArtifacts extends CachedArtifact {
     'ios-deploy',
   ];
 
+  static const Map<String, List<String>> _kExecutables = <String, List<String>>{
```

Comment:
Oh, maybe a comment here too, to give some context

Reply:
Done